.github/workflows/build_gem.yml

```diff
@@ -0,0 +1,24 @@
+name: Verify Gem Build
+
+on:
+  pull_request:
+    paths:
+      - 'dev/shopify_product_taxonomy.gemspec'
+      - 'dev/lib/**/*'
+      - '.github/workflows/build_gem.yml'
+
+jobs:
+  build:
+    name: Verify Gem Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - uses: ruby/setup-ruby@d5126b9b3579e429dd52e51e68624dda2e05be25 # v1.267.0
+        with:
+          ruby-version: '3.2'
```

Comment:
```suggestion

```

the Ruby version isn't needed (it'll take it from `.ruby-version`, same with the publish gem workflow)

Reply:
I think it should work with setting the `working-directory` for that action https://github.com/ruby/setup-ruby?tab=readme-ov-file#working-directory

```
      - uses: ruby/setup-ruby@d5126b9b3579e429dd52e51e68624dda2e05be25 # v1.267.0
        working-directory: dev
        with:
          bundler-cache: true
 ```